app/src/test/java/org/simple/clinic/activity/TheActivityControllerTest.kt

```diff
@@ -204,16 +204,16 @@ class TheActivityControllerTest {
   @Suppress("Unused")
   private fun `params for local user initial screen key`(): Array<Array<Any>> {
     return arrayOf(
-        arrayOf<Any>(true, NOT_LOGGED_IN, RegistrationPhoneScreen.KEY::class.java),
-        arrayOf<Any>(true, OTP_REQUESTED, HomeScreen.KEY::class.java),
-        arrayOf<Any>(true, RESET_PIN_REQUESTED, HomeScreen.KEY::class.java),
+        arrayOf<Any>(true, NOT_LOGGED_IN, RegistrationPhoneScreenKey()::class.java),
```

Comment:
You don't need to instantiate an object to get a class. `RegistrationPhoneScreenKey::class.java` should also work.

Reply:
Refactoring error